2018/07.md

```diff
@@ -72,7 +72,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     | timebox | topic | presenter |
     |:-------:|-------|-----------|
 
-1. Timeboxed overflow from previous meeting
```

Comment:
What about non-timeboxed overflow? We previously had two sections, so that timeboxed overflow would retain priority over non-timeboxed overflow. I see the "(overflow)" below; should we use an emoji (like the ⌛️ for late items) instead?

Reply:
I didn't see the non-timeboxed overflow section. I'd like to propose that we just put all of the overflow topics in one section, sorted in the order that they were sorted in the previous meeting's agenda, but not broken out into sub-categories.

Reply:
If you have a proposal for an overflow emoji, happy to switch to it. Looks like a nice way to present the information.

Reply:
Perhaos ➿ or ➰ or 🔁 or ♻️ or 🙉 or 🛅 or ⏪ ?

What do you see as the advantage of having all overflow items in one section? Would all the non-timeboxed ones have an empty "timebox" column?

Reply:
The advantage is, now that we have several different boxes, it would get unwieldy to have so many different overflow sections. It's just extra syntactic overhead that doesn't serve us, since people will not be adding their own new topics and figuring out where they go.